repos/system_upgrade/common/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -145,7 +146,19 @@ def _get_repositories_mapping(target_pesids):
         peseventsscanner_repomap.DEFAULT_PESID[get_target_major_version()], None
     )
     if not representative_repo:
-        api.current_logger().warning('Cannot determine the base target repository')
+        api.current_logger().warning('Cannot determine the base, representative, target repository.')
+        api.current_logger().info(
+            'Fallback: Create the artificial representative PESIDRepositoryEntry for the repository mapping'
```

Comment:
```suggestion
            'Fallback: Create a artificial representative PESIDRepositoryEntry for the repository mapping'
```

Reply:
> 

```suggestion
            'Fallback: Create an artificial representative PESIDRepositoryEntry for the repository mapping'
```

Sorry, just nitpicking the nitpick :)

Reply:
lol yeah

Reply:
done